addOns/ascanrules/CHANGELOG.md

```diff
@@ -4,6 +4,10 @@ All notable changes to this add-on will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/).
 
 ## Unreleased
+
```

Comment:
This empty line could be removed.

Reply:
Okay, working on that. I saw in my email that you said something about the httpMessage, but can't find the comment here, is this httpMessage change still needed?

Reply:
I removed it after I saw the full diff, I had been looking at a subset when I commented.